sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/aggregate/PercentileSuite.scala

```diff
@@ -84,6 +86,59 @@ class PercentileSuite extends SparkFunSuite {
     }
   }
 
+  test("class Percentile with frequency, high level interface, update, merge, eval...") {
```

Comment:
This test is almost a duplicate of the previous test. Lets just merge the two, and have a case without a frequency. I do like the parameterization.